Source/Urho3D/LuaScript/pkgs/Graphics/RenderPath.pkg

```diff
@@ -97,6 +97,8 @@ class RenderPath
     bool Append(XMLFile* file);
 
     void SetEnabled(const String tag, bool active);
+    bool IsEnabled(const String& tag) const;

```

Comment:
Lua binding has a quirk. It cannot use String reference.

Reply:
Really? I used existing bindings as example.. And to verify i ran a search for `const String&` on `LuaScript/pkgs` dir and there are many results. Are all those other bindings broken too?

Reply:
Most probably. I just did a search and there are 57 in total. Compared to 500+ that don't use reference. I suppose the issue that it may cause is quite subtle.